### DISCUSSION TITLE
We need to improve block support for classic themes

### DISCUSSION THREAD

**Discussion by carolinan** [WordPress/gutenberg#50080] [General]
First, I believe that block themes or using blocks for all parts of a site is the future. I have no interest in “hanging on to the past” and us* spending time on improving classic themes except for one important reason: The user experience.

(* Us as in WordPress contributors.)

I am worried that if a user, who is currently using a classic theme, has a bad experience using blocks, they will have very little reason and motivation to switch to using blocks.

If you are a user trying to, for example, change the color of a quote block and none of the color settings work, that doesn’t instill confidence, and it doesn’t encourage that user to use more blocks.

We need to give users an experience that says:
“Wow, this feature made it easier for me to use WordPress.”
“This helps me work faster.”
“Cool, I can change that setting and adjust the borders, too; this is fun.”

More of “I understand how I can use blocks to change my site. I want to use blocks for other parts like my site footer and product page.”
And less of “This setting is not working. It is frustrating, and it makes me second guess my decision to use WordPress because it is hard.“.

I worry that the direction right now is more of “Let’s starve classic themes out” than creating features that encourage people to use blocks no matter what type of theme they are using.

In the example, I mentioned the quote block.
A lot of the responsibility for solving similar issues is on the theme developer.
But it is increasingly difficult for a theme developer who tries to stay up to date with block theme changes, frequent Gutenberg changes, and also maintain existing classic themes.
(Did you, for example, know that one of the most popular free, classic themes in the theme directory asks users to uninstall Gutenberg?)

For theme developers, one issue is that classic themes and block themes load different CSS, and it is too much to keep track of.

We need to make it easier for both the user and the developer.
When it comes to block features: not “site building features,” classic themes need to be on par with block themes. Including spacing, color, and layout settings that are missing.

Improvements for classic theme users should be intentional.
When we add new block features, there should be a documented plan for if a feature should be available in classic themes, hybrid themes (a theme using PHP template files and theme.json), or block themes, including why, and preferably with a consequence analysis.

**Comment by carolinan** [comment]
Another example:
https://github.com/WordPress/gutenberg/issues/50249

**Comment by maurodf0** [comment]
Hi, 
this is my first comment in all Wordpress area, so might be useless, but i'll try ahah. BTW, sorry for my english, i'll try my best.

As beginner theme developer, i'm currentrly using the classic theme for build project, but i also agree with u, the blocks (are they into a classic/hybrid or block theme) must work fine everywhere, expecially for the theme into the repository.

I think that the hybrid theme (Php theme with theme.json) responde to this problem, cause you can build your theme but also give the user to style the blocks using the site editor, but for the developer (expecially the bigger one) this is very time consuming, cause seems that he has to build the theme two times, one with the .css style and the second one with the theme.json.

I think that at least for hybrid theme there have to be a documentation for expand all the functions of a classic theme and support all the new features of a block themes.
